huaweicloud/services/fgs/resource_huaweicloud_fgs_trigger.go

```diff
@@ -830,3 +837,18 @@ func resourceFunctionGraphTriggerDelete(d *schema.ResourceData, meta interface{}
 	d.SetId("")
 	return nil
 }
+
+func parseRequestError(respErr error) error {
+	var apiErr trigger.Error
+	if errCode, ok := respErr.(golangsdk.ErrDefault500); ok && errCode.Body != nil {
+		pErr := json.Unmarshal(errCode.Body, &apiErr)
+		if pErr == nil && apiErr.Code == "FSS.0500" && apiErr.Message == "Error getting associated function" {
```

Comment:
pls remove `apiErr.Message == "Error getting associated function"`, message maybe change.

Reply:
The errors of FSS.0500 code is not only `function not found`.